### PR TITLE
(MAINT) Update RHEL support for GCE

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Updated RHEL tests and suppoort as discovered when testing the GCE
patches in beaker-google.

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
Updates based on #20 and #1462

**Comment by trevor-vaughan** [comment]
@kevpl Revert all `rhel` additions per discussions

**Comment by pinkypie** [comment]
@kevpl @puppetlabs/beaker can I get another set of eyes on this one? There seems to be a good amount of history in these changes.

**Comment by trevor-vaughan** [comment]
@puppetlabs/beaker Can someone let me know what is broken in Jenkins?

**Comment by trevor-vaughan** [comment]
@mchllweeks Any updates here?

**Comment by kevpl** [comment]
@trevor-vaughan huh, it looks like something didn't go through on the rebase? Or perhaps we merged something after the rebase? Still has conflicts

**Comment by trevor-vaughan** [comment]
@kevpl So, I've performed the rebase but the spec tests are failing and I don't know why. I think I need some help on this one.

File: lib/beaker/host_prebuilt_steps.rb [hunk 1]
```diff
@@ -561,16 +561,9 @@ def set_env host, opts
...
-          if host['platform'] =~ /windows/ and host.is_cygwin?
-            env['CYGWIN'] = 'nodosfilewarning'
-          end
-          host.ssh_permit_user_environment
-          host.ssh_set_user_environment(env)
-        else
-          logger.debug(skip_msg)
+        unless skip_msg.nil?
+            logger.debug( skip_msg )
+            next
```
Comment (kevpl) [inline lib/beaker/host_prebuilt_steps.rb]: @trevor-vaughan it looks like the spec failures are caused by the fact that although it was a great idea to put the guard clause here first & get the error case out of the way, the main code path has been erased when I assume it should be just below the guard clause.

Comment (trevor-vaughan) [reply lib/beaker/host_prebuilt_steps.rb]: @kevpl Ah, I see. I won't have time to get to this before Sunday so feel free to just update the PR directly (I have allow maintainers to update) if you want to move it forward.

**Comment by kevpl** [comment]
last comment still current

**Comment by kevpl** [comment]
last comment still current

**Comment by kevpl** [comment]
last comment still current

**Comment by trevor-vaughan** [comment]
@kevpl Updated. You were spot on on the code segment.

**Comment by kevpl** [comment]
passed spec testing! these failures are unrelated & I believe I've helped them in another PR (#1546), so I'll re-kick these:

retest this please

**Comment by kevpl** [comment]
failures unrelated to changes

**Pull request merged**